File: Print First and Last Element/PrintFirstAndLastElement.java
public class PrintFirstAndLastElement {
    public static void main(String[] args) {
        java.util.Scanner sc = new java.util.Scanner(System.in);
        System.out.print("Enter the number of elements: ");
        int n = sc.nextInt();
        int[] a = new int[n];
        for (int i = 0; i < n; i++) {
            System.out.print("Enter element " + (i + 1) + ": ");
            a[i] = sc.nextInt();
        }
        System.out.println("First element: " + a[0]);
        System.out.println("Last element: " + a[n - 1]);
    }
}

File: Print First and Last Element/print_first_and_last_element.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int a[n];
    for (int i = 0; i < n; i++) {
        printf("Enter element %d: ", i + 1);
        scanf("%d", &a[i]);
    }
    printf("First element: %d\n", a[0]);
    printf("Last element: %d\n", a[n - 1]);
    return 0;
}
